templates/components/account/payment-methods-list.html

```diff
@@ -0,0 +1,50 @@
+{{#unless customer.payment_methods}}
+    {{> components/common/alert-info (lang 'account.payment_methods.no_methods')}}
+{{/unless}}
+
+<ul class="paymentMethodList">
+    {{#each customer.payment_methods}}
+        <h4 class="paymentMethodName">{{provider.display_name}}</h4>
+        {{#each methods}}
+        <li class="paymentMethod">
+            <div class="panel panel--paymentMethod">
+                <div class="panel-body">
+                    <div class="paymentMethod-card">
+                        {{#if brand '===' 'american_express'}}
+                            <img class="paymentMethod-icon" src="{{cdn 'img/payment-methods/american-express.svg'}}" alt="{{lang 'account.payment_methods.card_types.american_express'}}" title="{{lang 'account.payment_methods.card_types.american_express'}}">
```

Comment:
Can we simplify this `if .. else if .. else` structure. Can't we make the brand dynamic in the html.